the test error should not related to my PR
```
2026-06-29T03:57:46.160Z	ERROR	Error	{"flow": "init", "task": "Deploying BackupEntry for ETCD data", "error": "retry failed with context deadline exceeded, last error: condition not yet met: failed getting extensions.gardener.cloud/v1beta1.BackupEntry resource: backupentries.extensions.gardener.cloud \"kube-system--96a10972-9f7c-479d-b572-306bed03a749\" not found"}
github.com/gardener/gardener/pkg/utils/flow.(*execution).runNode.func2
	github.com/gardener/gardener/pkg/utils/flow/flow.go:251
2026-06-29T03:57:46.160Z	INFO	Finished	{"flow": "init"}
Error: 1 error occurred:
	* task "Deploying BackupEntry for ETCD data" failed: retry failed with context deadline exceeded, last error: condition not yet met: failed getting extensions.gardener.cloud/v1beta1.BackupEntry resource: backupentries.extensions.gardener.cloud "kube-system--96a10972-9f7c-479d-b572-306bed03a749" not found
make[1]: *** [Makefile:300: gind-up] Error 1
make[1]: Leaving directory '/home/prow/go/src/github.com/gardener/gardener'
18 skipped lines...
make[1]: Entering directory '/home/prow/go/src/github.com/gardener/gardener'
/home/prow/go/src/github.com/gardener/gardener/dev-setup/seed.sh down
Error from server (Forbidden): nodes is forbidden: User "system:serviceaccount:test-pods:gardener-prow-gobuildcache-ro" cannot list resource "nodes" in API group "" at the cluster scope
make[1]: *** [Makefile:345: seed-down] Error 1
make[1]: Leaving directory '/home/prow/go/src/github.com/gardener/gardener'
make: *** [Makefile:418: ci-e2e-kind-gardenadm-unmanaged-infra] Error 2
```

--- a/pkg/component/observability/plutono/dashboards/shoot/owners/worker/node-details-dashboard.json
+++ b/pkg/component/observability/plutono/dashboards/shoot/owners/worker/node-details-dashboard.json
@@ -1198,7 +1198,7 @@
       "dashLength": 10,
       "dashes": false,
       "datasource": null,
-      "description": "Reboot history for node $Node: bars show reboot events (detected after node rejoins cluster; one bar per scrape interval where a reboot is detected) on the left axis, line shows node uptime since last boot on the right axis.",
+      "description": "Reboot history for node $Node: line shows reboot events (detected after node rejoins cluster) on the left axis, line shows node uptime since last boot on the right axis.",
       "fieldConfig": {
         "defaults": {},
         "overrides": []
@@ -1237,9 +1237,9 @@
       "seriesOverrides": [
         {
           "alias": "Reboots",
-          "bars": true,
+          "bars": false,
           "color": "#F2495C",
-          "lines": false,
+          "lines": true,
           "yaxis": 1
         },
         {
@@ -1254,9 +1254,9 @@
       "steppedLine": false,
       "targets": [
         {
-          "expr": "(\n  last_over_time(node_boot_time_seconds{node=~\"$Node\"}[1h])\n  !=\n  last_over_time(node_boot_time_seconds{node=~\"$Node\"}[1h] offset 1m)\n) > bool 0",
+          "expr": "changes(node_boot_time_seconds{node=~\"$Node\"}[$__rate_interval])",
           "format": "time_series",
-          "interval": "",
+          "interval": "1m",
           "intervalFactor": 1,
           "legendFormat": "Reboots",
           "refId": "A"
@@ -1607,7 +1607,7 @@
     ]
   },
   "time": {
-    "from": "now-30m",
+    "from": "now-3h",
     "to": "now"
   },
   "timepicker": {

--- a/pkg/component/observability/plutono/dashboards/shoot/owners/worker/node-details-dashboard.json
+++ b/pkg/component/observability/plutono/dashboards/shoot/owners/worker/node-details-dashboard.json
@@ -1198,7 +1198,7 @@
       "dashLength": 10,
       "dashes": false,
       "datasource": null,
-      "description": "Reboot history for node $Node: bars show reboot events (detected after node rejoins cluster; each reboot shown once per 10-minute interval) on the left axis, line shows node uptime since last boot on the right axis.",
+      "description": "Reboot history for node $Node: bars show reboot events (detected after node rejoins cluster; one bar per scrape interval where a reboot is detected) on the left axis, line shows node uptime since last boot on the right axis.",
       "fieldConfig": {
         "defaults": {},
         "overrides": []
@@ -1254,7 +1254,7 @@
       "steppedLine": false,
       "targets": [
         {
-          "expr": "changes(node_boot_time_seconds{node=~\"$Node\"}[10m:10m])",
+          "expr": "(\n  last_over_time(node_boot_time_seconds{node=~\"$Node\"}[1h])\n  !=\n  last_over_time(node_boot_time_seconds{node=~\"$Node\"}[1h] offset 1m)\n) > bool 0",
           "format": "time_series",
           "interval": "",
           "intervalFactor": 1,

--- a/pkg/component/observability/plutono/dashboards/shoot/owners/worker/node-pool-dashboard.json
+++ b/pkg/component/observability/plutono/dashboards/shoot/owners/worker/node-pool-dashboard.json
@@ -1189,7 +1189,7 @@
       "dashLength": 10,
       "dashes": false,
       "datasource": null,
-      "description": "Reboot events per node over time (detected after node rejoins cluster; each reboot shown once per 10-minute interval). Filtered by selected worker group(s).",
+      "description": "Reboot events per node over time (detected after node rejoins cluster; one bar per scrape interval where a reboot is detected). Filtered by selected worker group(s).",
       "fieldConfig": {
         "defaults": {},
         "overrides": []
@@ -1232,7 +1232,7 @@
       "steppedLine": false,
       "targets": [
         {
-          "expr": "changes(\n  (\n      node_boot_time_seconds\n    + on (node) group_left (label_worker_gardener_cloud_pool)\n      0 * kube_node_labels{label_worker_gardener_cloud_pool=~\"$worker_group\"}\n  )[10m:10m]\n)",
+          "expr": "(\n  last_over_time(node_boot_time_seconds[1h])\n  + on (node) group_left (label_worker_gardener_cloud_pool)\n    0 * kube_node_labels{label_worker_gardener_cloud_pool=~\"$worker_group\"}\n)\n!=\n(\n  last_over_time(node_boot_time_seconds[1h] offset 1m)\n  + on (node) group_left (label_worker_gardener_cloud_pool)\n    0 * kube_node_labels{label_worker_gardener_cloud_pool=~\"$worker_group\"}\n)",
           "format": "time_series",
           "interval": "",
           "intervalFactor": 1,

--- a/pkg/component/observability/plutono/dashboards/shoot/owners/worker/node-pool-dashboard.json
+++ b/pkg/component/observability/plutono/dashboards/shoot/owners/worker/node-pool-dashboard.json
@@ -1185,11 +1185,11 @@
     },
     {
       "aliasColors": {},
-      "bars": true,
+      "bars": false,
       "dashLength": 10,
       "dashes": false,
       "datasource": null,
-      "description": "Reboot events per node over time (detected after node rejoins cluster; one bar per scrape interval where a reboot is detected). Filtered by selected worker group(s).",
+      "description": "Reboot events per node over time (detected after node rejoins cluster). Filtered by selected worker group(s).",
       "fieldConfig": {
         "defaults": {},
         "overrides": []
@@ -1215,7 +1215,7 @@
         "total": false,
         "values": true
       },
-      "lines": false,
+      "lines": true,
       "linewidth": 1,
       "nullPointMode": "null as zero",
       "options": {
@@ -1232,9 +1232,9 @@
       "steppedLine": false,
       "targets": [
         {
-          "expr": "(\n  last_over_time(node_boot_time_seconds[1h])\n  + on (node) group_left (label_worker_gardener_cloud_pool)\n    0 * kube_node_labels{label_worker_gardener_cloud_pool=~\"$worker_group\"}\n)\n!=\n(\n  last_over_time(node_boot_time_seconds[1h] offset 1m)\n  + on (node) group_left (label_worker_gardener_cloud_pool)\n    0 * kube_node_labels{label_worker_gardener_cloud_pool=~\"$worker_group\"}\n)",
+          "expr": "changes(node_boot_time_seconds[$__rate_interval])\n* on (node) group_left (label_worker_gardener_cloud_pool)\n  kube_node_labels{label_worker_gardener_cloud_pool=~\"$worker_group\"}",
           "format": "time_series",
-          "interval": "",
+          "interval": "1m",
           "intervalFactor": 1,
           "legendFormat": "{{node}}",
           "refId": "A"
@@ -1444,7 +1444,7 @@
     ]
   },
   "time": {
-    "from": "now-30m",
+    "from": "now-3h",
     "to": "now"
   },
   "timepicker": {

